airbyte-cdk/bulk/core/extract/src/main/kotlin/io/airbyte/cdk/check/CheckOperation.kt

```diff
@@ -34,37 +37,15 @@ class CheckOperation<T : ConfigurationJsonObjectBase>(
             val config: SourceConfiguration = configFactory.make(pojo)
             log.info { "Connecting for config check." }
             metadataQuerierFactory.session(config).use { connectionCheck(it) }
-        } catch (e: SQLException) {
-            log.debug(e) { "SQLException while checking config." }
-            val message: String =
-                listOfNotNull(
-                        e.sqlState?.let { "State code: $it" },
-                        e.errorCode.takeIf { it != 0 }?.let { "Error code: $it" },
-                        e.message?.let { "Message: $it" },
-                    )
-                    .joinToString(separator = "; ")
-            ApmTraceUtils.addExceptionToTrace(e)
-            outputConsumer.accept(
-                AirbyteErrorTraceMessage()
-                    .withFailureType(AirbyteErrorTraceMessage.FailureType.CONFIG_ERROR)
-                    .withMessage(message)
-                    .withInternalMessage(e.toString())
-                    .withStackTrace(ExceptionUtils.getStackTrace(e)),
-            )
-            outputConsumer.accept(
-                AirbyteConnectionStatus()
-                    .withMessage(message)
-                    .withStatus(AirbyteConnectionStatus.Status.FAILED),
-            )
-            log.info { "Config check failed." }
-            return
```

Comment:
All this `SQLException` logic had no business being in the general-purpose `CheckOperation` anyway. It's been moved to `JdbcExceptionClassifier` in the JDBC toolkit, where it belongs.